File: pl-domain/religion-service/src/main/java/com/polylab/religion/ReligionServiceApplication.java
package com.polylab.religion;

import com.jxareas.jxcore.annotations.EnableJxCore;
import com.jxareas.jxcore.annotations.EnableNonRestrictiveCorsPolicy;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@EnableJxCore
@EnableNonRestrictiveCorsPolicy
@SpringBootApplication
public class ReligionServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(ReligionServiceApplication.class, args);
    }

}

File: pl-domain/religion-service/src/main/java/com/polylab/religion/ReligionServiceApplication.kt
package com.polylab.religion;

import com.jxareas.efficax.annotations.EnableEfficax
import com.jxareas.efficax.annotations.EnableNonRestrictiveCorsPolicy
import com.jxareas.efficax.extensions.with
import org.springframework.boot.autoconfigure.SpringBootApplication

@EnableEfficax
@EnableNonRestrictiveCorsPolicy
@SpringBootApplication
open class ReligionServiceApplication

fun main(vararg args : String) = ReligionServiceApplication::class.with(*args)
